tools/cli/internal/openapi/file.go

```diff
@@ -29,27 +29,34 @@ import (
 const (
 	JSON    = "json"
 	YAML    = "yaml"
+	ALL     = "all"
```

Comment:
[No need to address the comment]. Something to consider in the future is tohave the --format flag as a list so that we can provide multiple formats. The drawback of having ALL are:

- it is not clear what ALL traslate to by looking at the command but we force users to run --help
- if we add support for another format (not likely), ALL will start generating other format too 🤔 

I think is fine to have all since we are the user of this cli and it is not likely we will support other formats

Reply:
ack, if we add more formats we can defintely revisit! or keep supporitng the all to generate all formats